README.md

```diff
@@ -364,6 +370,21 @@ To start any container separately:
 
 ### Users & Authentication
 
+Set the `KEYCLOAK_SERVER_URL` and `KEYCLOAK_CLIENT_ID` environment variables if
+you want to use Keycloak as authentication server.
+`KEYCLOAK_CLIENT_ID` (defaults to `aether`) is the public client that allows
+the aether module to authenticate using the Keycloak REST API.
+This client id must be added to all the realms used by the aether module.
```

Comment:
It is required to create the `aether` client in each realm and include KEYCLOAK_CLIENT_ID in a production instance that doesn't use basic_auth?

Reply:
Because we are using keycloak "behind the scenes" we need a client to get the user token, the client can be different for each module I use the same in order to simplify the development, but you could have one for kernel and another for the rest... :woman_shrugging: 

```
curl --data \
    "grant_type=password&client_id={CLIENT}&username={user}&password={password}" \
    http://localhost:8080/auth/realms/{REALM}/protocol/openid-connect/token`
```

This is the one of the possible implementation, the other one is going to keycloak, authenticate there and take the cookie from the response...